.github/workflows/test.yml

```diff
@@ -13,19 +13,26 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - 3.6
-          - 3.7
-          - 3.8
-          - 3.9
-          - pypy3
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "pypy-3.7"
+          - "pypy-3.8"
         tox-environment:
           - django22-alchemy-mongoengine
           - django31-alchemy-mongoengine
           - django32-alchemy-mongoengine
         include:
-          - python-version: 3.9
+          - python-version: "pypy-3.8"
+            tox-environment: django40-alchemy-mongoengine
+          - python-version: "3.8"
+            tox-environment: django40-alchemy-mongoengine
+          - python-version: "3.9"
+            tox-environment: django40-alchemy-mongoengine
+          - python-version: "3.10"
+            tox-environment: django40-alchemy-mongoengine
```

Comment:
Instead of expanding the matrix, perhaps add `django40-alchemy-mongoengine` to the `tox-environment` list and make python `3.7` a special case (that only runs `django22-alchemy-mongoengine` and `django32-alchemy-mongoengine`)?

Reply:
Sure, makes sense!